.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Create a report to help us improve the library
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Before raising, please check if somebody else has already reported your issue. -->
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### Steps to reproduce
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Environment
+* Android Studio version:
+* Android Gradle version:
+* Bugsnag manifest section (if modified):
```

Comment:
Gradle Wrapper version would be useful to collect too as this can affect how the plugin works quite a bit.